pkg/render/kubecontrollers/kube-controllers.go

```diff
@@ -100,6 +103,9 @@ type KubeControllersConfiguration struct {
 
 	// List of namespaces that are running a kube-controllers instance that need a cluster role binding.
 	BindingNamespaces []string
+
+	// Whether or not to run the rendered components in multi-tenant mode.
```

Comment:
This comment isn't correct - the Tenant object provides tenant configuration for both single and multi-tenant modes. If this is nil, then we should run in zero-tenant mode.